components/common/Table.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
```

Comment:
Let's rename this file and component to `ApplicationsTable` and move it to /components/admin/